pkg/services/sqlstore/sqlstore.go

```diff
@@ -78,7 +78,7 @@ func ProvideService(cfg *setting.Cfg, cacheService *localcache.CacheService, bus
 }
 
 func ProvideServiceForTests(migrations registry.DatabaseMigrator) (*SQLStore, error) {
-	return initTestDB(migrations, InitTestDBOpt{EnsureDefaultOrgAndUser: true})
+	return initTestDB(migrations, InitTestDBOpt{EnsureDefaultOrgAndUser: true, Features: featuremgmt.WithFeatures()})
```

Comment:
Are these changes still needed? Maybe can revert changes in this file if not needed for you?